newIDE/app/src/ProjectsStorage/ResourceFetcher/BrowserResourceFetcher.js

```diff
@@ -6,6 +6,7 @@ import {
   type FetchAllProjectResourcesFunction,
 } from './index';
 import CloudStorageProvider from '../CloudStorageProvider';
+import { moveUrlResourcesToCloudFilesIfPrivate } from '../CloudStorageProvider/CloudResourceMover';
```

Comment:
You can see there is "something weird going on" because you're in a "ResourceFetcher" file and you import something from a "ResourceMover".

Moving !== Fetching.
The first one is to move existing resources from a project to another.
While fetching is to import resources stored somewhere (but not in a project) inside a project.